<p>This rule detects methods with too many params</p>
<h2>Noncompliant Code Example</h2>
<pre>
public void methodTest(int input1, int input2, int input3, int input4, int input5, int input6) { //Noncompliant {{TOO MANY PARAMS}}
		System.out.println(input1+input2+input3+input4+input5+input6);
	}
</pre>
<h2>Compliant Solution</h2>
<pre>
public void methodTest(int input1, int input2, int input3) {
		System.out.println(input1+input2+input3); 
</pre>
